data/processed/hesa/total_university_buildings.csv

```diff
@@ -0,0 +1,121 @@
+,year,nuts_id,nuts_year_spec,total_university_buildings
+0,2015/16,UKH2,2018,419.0
```

Comment:
I think `year` is supposed to be an `int` so we should probably choose the start or the end of the academic year? (*)